haystack/components/embedders/sentence_transformers_text_embedder.py

```diff
@@ -59,6 +60,9 @@ def __init__(
             If True shows a progress bar when running.
         :param normalize_embeddings:
             If True returned vectors will have length 1.
+        :param trust_remote_code:
+            If trust_remote_code is false only HuggingFace verified model architectures are allowed. If true
+            then custom models and scripts are allowed.
```

Comment:
Please also update this.

Reply:
Done with [2bad887](https://github.com/deepset-ai/haystack/pull/7356/commits/2bad88762608a591fc11488d620457eb23ef8951)